packages/wonder-blocks-core/src/util/add-style.tsx

```diff
@@ -22,18 +40,19 @@ export default function addStyle<
             processStyleList([reset, defaultStyle, style]);
 
         return (
-            // @ts-expect-error [FEI-5019] - TS2604 - JSX element type 'Component' does not have any construct or call signatures.
+            // @ts-expect-error: TS says this is not assignable to the return forwardRef()'s return type.
+            // Type 'Omit<PropsWithChildren<Props>, "style" | "className"> & { ref: ForwardedRef<ElementMap[T]>; className: string; style: CSSProperties; }' is not assignable to type 'IntrinsicAttributes & LibraryManagedAttributes<T, SVGProps<SVGSymbolElement> & ClassAttributes<HTMLObjectElement> & ... 179 more ... & SVGProps<...>>'.
+            //   Type 'Omit<PropsWithChildren<Props>, "style" | "className"> & { ref: ForwardedRef<ElementMap[T]>; className: string; style: CSSProperties; }' is not assignable to type 'LibraryManagedAttributes<T, SVGProps<SVGSymbolElement> & ClassAttributes<HTMLObjectElement> & ObjectHTMLAttributes<HTMLObjectElement> & ... 178 more ... & SVGProps<...>>'
```

Comment:
I'm not sure how helpful having this error message here is, but I'm going to leave it just case people are curious why the types aren't compatible.